Makefile

```diff
@@ -473,7 +473,7 @@ KUSTOMIZE_VERSION ?= v5.0.3
 CONTROLLER_TOOLS_VERSION ?= v0.16.1
 GOLANGCI_LINT_VERSION ?= v1.57.2
 KIND_VERSION ?= v0.20.0
-CHAINSAW_VERSION ?= v0.2.8
+CHAINSAW_VERSION ?= v0.2.11
```

Comment:
this version does not work on go 1.22. I would wait with updating before we upgrade the go version

Reply:
If that's a concern for us, we should install it by downloading the release binary instead. Our supported Go version shouldn't get in the way of tooling upgrades.